### PR TITLE
fix: albums view acc to gallery standard

### LINKED ISSUE #2931: Album View Layout Change and font size change
**Describe the bug**
Currently the album layout view hierachy is more complex, change that to constraint layout
Also, handling the font size in the textViews according to gallery of android phone.

Issues #2921 

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by codedsun**
Fixed #2931 

Changes: 
- Follows android standard gallery 
- Font size and style set to normal
- Layout hierarchy changed to Constraint and simple one
- UI improved

Before Screenshot
ezgif com-video-to-gif-2

Screenshots of the change: 

Screenshot_2019-11-23-01-05-49-92_e9fa3137047667733754e6d84fae6a64

ezgif com-video-to-gif

### CODE REVIEW

**Comment by codedsun** [comment]
@mariobehling Please check the UI.

@yashk2000 Please check

**Comment by codedsun** [comment]
please chck @yashk2000

**Comment by codedsun** [comment]
@iamareebjamal Please get this done

**Comment by codedsun** [comment]
@iamareebjamal process this also

File: app/src/main/java/org/fossasia/phimpme/gallery/adapters/AlbumsAdapter.java [hunk 1]
```diff
@@ -35,8 +33,9 @@
 import org.fossasia.phimpme.gallery.util.ColorPalette;
 import org.fossasia.phimpme.gallery.util.PreferenceUtil;
 import org.fossasia.phimpme.gallery.util.ThemeHelper;
+import org.jetbrains.annotations.NotNull;
 
-/** Created by dnld on 1/7/16. */
+/** Created by codedsun on 23/11/19. */
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/phimpme/gallery/adapters/AlbumsAdapter.java]: Remove this line

**Comment by iamareebjamal** [comment]
UI changes go through @mariobehling 

Also, for any UI changes, ALWAYS put before and after images

**Comment by codedsun** [comment]
@iamareebjamal Okay

**Comment by codedsun** [comment]
@mariobehling Please check this. Thanks

**Comment by iamareebjamal** [comment]
Add before and after screenshots

**Comment by codedsun** [comment]
@iamareebjamal Uploading in few mins

**Comment by codedsun** [comment]
> @iamareebjamal Uploading in few mins

Updated

**Comment by codedsun** [comment]
@mariobehling Please check the UI

**Comment by codedsun** [comment]
@mariobehling Please check this for UI perspective.

**Comment by codedsun** [comment]
@mariobehling Please finalise this

**Comment by codedsun** [comment]
@mariobehling Please check this

**Pull request merged**